content/docs/pipelines/sdk/install-sdk.md

```diff
@@ -73,7 +73,17 @@ up Python using [Miniconda](https://conda.io/miniconda.html):
 Run the following command to install the Kubeflow Pipelines SDK:
 
 ```bash
-pip install https://storage.googleapis.com/ml-pipeline/release/latest/kfp.tar.gz --upgrade
+pip3 install kfp --upgrade
+```
+*Note that if you are not using a virutal envirometn such as conda in step 1, you may recieve a error during installation as follows: 
```

Comment:
Suggested revisions - spelling, formatting, etc: 

**Note:** If you are not using a virtual environment, such as `conda`, when installing the Kubeflow Pipelines SDK, you may receive the following error:

Reply:
done